src/main/java/org/jabref/gui/preferences/ExternalTab.java

```diff
@@ -197,24 +215,19 @@ public Node getBuilder() {
         return builder;
     }
 
-    private void addSettingsButton(final PushToApplication application, GridPane panel, int index) {
-        PushToApplicationSettings settings = frame.getPushApplications().getSettings(application);
-        Button button = new Button(Localization.lang("Settings for %0", application.getApplicationName()));
-        button.setPrefSize(150, 20);
-        button.setOnAction(e -> PushToApplicationSettingsDialog.showSettingsDialog(dialogService, settings, index));
-        if ((index % 2) == 0) {
-            panel.add(button, 1, (index / 2) + 1);
-        } else {
-            panel.add(button, 2, (index / 2) + 1);
-        }
-    }
-
     @Override
     public void setValues() {
 
         emailSubject.setText(prefs.get(JabRefPreferences.EMAIL_SUBJECT));
         openFoldersOfAttachedFiles.setSelected(prefs.getBoolean(JabRefPreferences.OPEN_FOLDERS_OF_ATTACHED_FILES));
 
+        String pushToApplicationName = prefs.get(JabRefPreferences.PUSH_TO_APPLICATION);
+        List<PushToApplication> pushToApplications = frame.getPushToApplicationsManager().getApplications();
+        pushToApplicationComboBox.setValue(
+                pushToApplications.stream()
```

Comment:
Maybe extract this logic to a method `getActivePushToApplication(PushToApplicationsManager manager)` in `JabRefPreferences`?